docs/website/docs/building-from-source/getting-started.md

```diff
@@ -70,66 +67,75 @@ cd iree
 git submodule update --init
 ```
 
-Configure then build all targets using CMake:
+The most basic CMake workflow is:
+
+``` shell
+# Configure
+cmake -G Ninja -B ../iree-build/ .
+
+# Build
+cmake --build ../iree-build/
+```
+
+!!! Caution "Caution - slow builds"
+    The compiler build is complex. You will want a powerful machine and to tune
```

Comment:
If you want, you could give some idea of the expected build time. Depending on the context, "slow" could mean 12 minutes or 12 hours.

Reply:
"As of March, 2023, expect that a full compiler build on a 64 core Linux machine with good I/O may take 5-7 minutes. On MacOS, building non-universal on an M1 Mini should take around 15 minutes."